ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -539,6 +578,8 @@ func (s *LifeCycleServiceServer) Init(ctx context.Context, in *pb.InitRequest) (
 		}
 	}
 
+	checkIdpfNetDevices(s.mode)
```

Comment:
Add error condition in checkIdpfNetDevices  has len of  pfList   zero after rmmod/insmod to bringup idpf interfaces.

Reply:
@vkannapp Hi Vadivel, you mean...after rmmod/insmod...if pflist is 0, to throw an error message.

Reply:
Right!  Error condition if   IPU/idpf interfaces doesn't come up.